x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.ts

```diff
@@ -43,15 +96,59 @@ interface RelevanceTuningValues {
   query: string;
   unsavedChanges: boolean;
   dataLoading: boolean;
-  searchResults: object[] | null;
+  searchResults: Result[] | null;
   resultsLoading: boolean;
 }
 
+const UPDATE_SUCCESS_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.updateSuccess',
+  {
+    defaultMessage: 'Relevance successfully tuned. The changes will impact your results shortly.',
+  }
+);
+const DELETE_SUCCESS_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.deleteSuccess',
+  {
+    defaultMessage:
+      'Relevance has been reset to default values. The change will impact your results shortly.',
+  }
+);
+const RESET_CONFIRMATION_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.resetConfirmation',
+  {
+    defaultMessage: 'Are you sure you want to restore relevance defaults?',
+  }
+);
+const DELETE_CONFIRMATION_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.relevanceTuning.messages.deleteConfirmation',
+  {
+    defaultMessage: 'Are you sure you want to delete this boost?',
+  }
+);
+
 // If the user hasn't entered a filter, then we can skip filtering the array entirely
 const filterIfTerm = (array: string[], filterTerm: string): string[] => {
   return filterTerm === '' ? array : array.filter((item) => item.includes(filterTerm));
 };
 
+const removeBoostStateProps = (searchSettings: SearchSettings) => {
+  const updatedSettings = cloneDeep(searchSettings);
+  const { boosts } = updatedSettings;
+  const keys = Object.keys(boosts);
+  keys.forEach((key) => boosts[key].forEach((boost) => delete boost.newBoost));
+
+  return updatedSettings;
+};
+
+const parseBoostCenter = (fieldType: SchemaTypes, value: string | number) => {
+  // Leave non-numeric fields alone
+  if (fieldType === NUMBER) {
+    const floatValue = parseFloat(value as string);
+    return isNaN(floatValue) ? value : floatValue;
+  }
+  return value;
+};
+
```

Comment:
Similar request for the above utility FNs - can we move them to a `utils.ts` file?

Reply:
Yes.

Reply:
e98afe4

Reply:
@constancecchen I ended up adding separate tests for these: 79a499ab3c3835ae3e8825d9636d56f41de7f121